pyrasterframes/src/main/python/pyrasterframes/rf_types.py

```diff
@@ -349,9 +349,102 @@ def __div__(self, right):
         return self.__truediv__(right)
 
     def dimensions(self):
-        # list of cols, rows as is conventional in GeoTrellis and RasterFrames
+        """ Return a list of cols, rows as is conventional in GeoTrellis and RasterFrames."""
         return [self.cells.shape[1], self.cells.shape[0]]
 
+    def to_png(self):
```

Comment:
I feel like this deserves a "separation of concerns" analysis.

My preference would be to have a separate module that is optionally imported to either add `foo(Tile)` functions or inject extensions on `Tile` class, and keep this focused on the minimal representation of a `Tile` in Python.

Reply:
@metasim can you take another look. we retain the `_repr_png_` bound method, but that will (ostensibly) only be called by Ipython or jupyter;